tests/FSharpPlus.Tests/FSharpPlus.Tests.fsproj

```diff
@@ -5,7 +5,7 @@
     <FscToolExe>$(FSC_ExePathCompilerBuild)</FscToolExe>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
```

Comment:
net45, do you want net47 here?

Reply:
For a library, better to target lowest possible framework I think.

FSharp.Core targets net45.

Reply:
Yes, so are there benefits of testing on net45?